NEWS.md

```diff
@@ -6,20 +6,37 @@
-
-* bioRad is now fully available on Windows
-* Faster implementations of functions previously dependent on Docker, such as  `calculate_vp()` and `apply_mistnet()` and `read_pvolfile()`.
-* argument `local_install` in `calculate_vp()` and `apply_mistnet()` is now deprecated.
-* functions `check_docker()` and `update_docker()` have been deprecated
-* function `vol2bird_version()` has been migrated to package vol2birdR and can be accessed by `vol2birdR::vol2bird_version()`
-* fix bug in height integration, which excluded the lowest bins for certain low altitude radars (#534)
-* fast parallel mistnet runs (https://github.com/adokter/vol2birdR/issues/16)
-* simplify installation, including automatic installation of rhdf5 from bioconductor (#464)
-* new sep argument in read_vpts() (#536)
-* allow odim files with missing source attribute. Extraction of radar indentifer from what/source attribute in read_pvolfiles updated to function as read_vpfiles(), i.e. using the NOD identifier in the source attribute, if missing try RAD, if also missing try WMO, if nothing found use "unknown" (2f6935c).
-* new function `list_vpts_aloft()` produces a list of aloft archive urls for time series of vertical profiles (`vpts`). This list of urls can then be used to bulk download this data using any number of external tools. #553
+bioRad 0.7 includes a major backend overhaul that deprecates the use of Docker. All Docker-dependent functionality is now included in the new dependency package [vol2birdR](https://cran.r-project.org/package=vol2birdR) package, which needs to be installed as part of bioRad. All bioRad functions remain the same, but several functions will run considerable faster.
 
+# New features
+
+* bioRad is now fully available on Windows.
+
+* Simplified installation, including automatic installation of rhdf5 from bioconductor (#464).
+
+* Faster implementations of functions previously dependent on Docker, such as `calculate_vp()`, `apply_mistnet()` and `read_pvolfile()`.
+
+* Faster parallel mistnet runs (https://github.com/adokter/vol2birdR/issues/16).
+
+* New function `list_vpts_aloft()` produces a list of [aloft](https://aloftdata.eu/browse/) archive URLs for time series of vertical profiles (`vpts`). This list of URLs can then be used to bulk download data using any number of external tools (#553).
+
+* Updated function `read_vpts()` supports reading `vp`/`vpts` data in hdf5 and [VPTS CSV](https://aloftdata.eu/vpts-csv/) format.
+
+* New function `read_stdout()` replaces previous functionality of `read_vpts()` to read vol2bird stdout format. It also has a new `sep` argument (#536).
+
+* `read_pvolfiles()` now allows Odim files with missing `source` attribute. The functionality is similar to `read_vpfiles()`, i.e. using the NOD identifier in the source attribute, if missing try RAD, if also missing try WMO, if nothing found use `unknown` (2f6935c).
```

Comment:
Suggest changing `Odim` to `ODIM_H5`, which seems to be the official acronym now used. Latest format specification: https://www.eumetnet.eu/wp-content/uploads/2021/07/ODIM_H5_v2.4.pdf